automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -29,7 +29,8 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
   val projectName: String = Config.Projects.common
 
   // Unless otherwise declared, this auth token will be used for API calls.
-  implicit val authToken: AuthToken = AuthTokens.fred
+  val defaultUser: Credentials = UserPool.chooseCurator().head
```

Comment:
Should add a comment that the choice of curator here is only to avoid the chance of choosing the same user as the tests (which use auth domain users), not because we specifically need to use a curator.